packages/swap/src/event-handlers/v140/depositBoosted.ts

```diff
@@ -0,0 +1,44 @@
+import { z } from 'zod';
+import { ONE_IN_PIP } from '@/shared/functions';
+import { actionSchema, internalAssetEnum, number, u128 } from '@/shared/parsers';
+import { EventHandlerArgs } from '..';
+
+export const depositBoostedSchema = z.object({
+  asset: internalAssetEnum,
+  amounts: z.array(z.tuple([number, u128])),
+  ingressFee: u128,
+  boostFee: u128,
+  action: actionSchema,
+  // channelId: u128,
+  // depositDetails: cfChainsBtcUtxoId,
+  // prewitnessedDepositId: u128,
+});
+
+// DepositBoosted event is emitted instead of NetworkDepositReceived event in v140 due to boost
+// We need to update the depositAmount and store the ingress fee just like we do in the NetworkDepositReceived event
+export const depositBoosted = async ({ prisma, event }: EventHandlerArgs) => {
+  const { asset, boostFee, action, ingressFee, amounts } = depositBoostedSchema.parse(event.args);
+
+  if (action.__kind === 'Swap') {
+    const depositAmount = amounts.reduce((acc, [_, amount]) => acc + amount, BigInt(0));
```

Comment:
i think the deposit amount should be the amount before the boost fee (and ingress fee), right? 
i suspect that we need to add the boost fee to this to get that value, but i might be mistaken

Reply:
From what i saw in the events, the summed `amounts` array of tuples will hold the initial deposit amount.

Reply:
okay cool, nevermind then!